### ISSUE TITLE
No KDE5 theme in PyQt5 application

### ISSUE THREAD

**Issue by Insoleet** [pyinstaller/pyinstaller#1769]
Hello,

I'm building my application using the following command : 
```
pyinstaller src/sakia/main.py --additional-hooks-dir hook
```

When running the binary, the application is displaying a Qt5 theme instead of KDE theme : 

```
static QPlatformTheme* QKdeTheme::createKdeTheme(): Unable to determine KDE dirs
```

**Comment by bjones1** [comment]
What version of PyInstaller are you using? What OS?

**Comment by Insoleet** [comment]
I'm using archlinux and PyInstaller 3.1 from pip.

**Comment by bjones1** [comment]
Could you provide some help in diagnosing this? Per #1752, what files does KDE need to get it working? Looking at https://github.com/pyinstaller/pyinstaller/commit/03902c1f0dea7d521c282c904339ae4ec05f59fb, perhaps similar changes would work on PyQt5. Therefore, would you:

1. Try the latest version from git.
2. Try applying changes similar to the commit I referenced, to see if that helps?

**Comment by Insoleet** [comment]
Thanks for your help. I was able to find the problem, a qt5_plugin was missing from includes.

https://github.com/pyinstaller/pyinstaller/pull/1783

**Comment by bjones1** [comment]
Thanks! Closed by merging #1783.
